### PR TITLE
hexdump: Show chars even if not in 'byte' mode.

### LINKED ISSUE #396: Idea: always display ascii when hexdumping memory
Not really an issue, I was just wondering why don't we display the ascii view even when users type `hexdump (qword|dword|word) address`? 

Something like this:

hexdump_view

I think it's always useful to have that information + the offsets in memory. Imho that's the best view that you can have in memory, what do you think?

### PR DESCRIPTION
**Pull request by Grazfather**
Fixes #396 
1__tmux

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -7849,8 +7849,9 @@ def _hexdump(self, start_addr, length, arrange_as, offset=0):
             sym = "<{:s}+{:04x}> ".format(*sym) if sym else ""
             mem = read_memory(cur_addr, l)
             val = struct.unpack(fmt_pack, mem)[0]
+            text = "".join([chr(b) if 0x20 <= b < 0x7F else "." for b in mem])
```
Comment (hugsy) [inline gef.py]: This will not work as intended on GDB + Python 2

**Comment by hugsy** [comment]
@Grazfather Although I have nothing against this request, I would prefer if showing the text when not in byte mode is only done is a specific setting is set to `true`. Do you want me to do that ?

File: gef.py (hunk 1 above)
Comment (Grazfather) [reply gef.py]: Why not? I took it from the hexdump command.

**Comment by Grazfather** [comment]
Go for it.

**Pull request merged**